test/box-luatest/gh_10180_upgrade_from_1_6_9_test.lua

```diff
@@ -0,0 +1,29 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new({
+        datadir = 'test/box-luatest/upgrade/1.6.9',
+    })
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+g.test_upgrade = function(cg)
+    cg.server:exec(function()
+        t.assert_equals(box.space._schema:get{'version'}, {'version', 1, 6, 8})
```

Comment:
The test is called 'upgrade from 1.6.9', which means the schema must be at least 1.6.9, not 1.6.8, no? Or did 1.6.9 not have any own changes since 1.6.8 and hence didn't have its own schema version?

Reply:
> Or did 1.6.9 not have any own changes since 1.6.8 and hence didn't have its own schema version?

Yep:

```lua
./src/tarantool: version 1.6.9-94-ga5da60bdd
type 'help' for interactive help
tarantool> box.cfg{}
2024-10-15 17:10:21.604 [143921] main/101/interactive C> version 1.6.9-94-ga5da60bdd
2024-10-15 17:10:21.604 [143921] main/101/interactive C> log level 5
2024-10-15 17:10:21.604 [143921] main/101/interactive I> mapping 1073741824 bytes for tuple arena...
2024-10-15 17:10:21.606 [143921] main/101/interactive I> initializing an empty data directory
2024-10-15 17:10:21.607 [143921] snapshot/101/main I> creating `./00000000000000000000.snap.inprogress'
2024-10-15 17:10:21.607 [143921] snapshot/101/main I> saving snapshot `./00000000000000000000.snap.inprogress'
2024-10-15 17:10:21.607 [143921] snapshot/101/main I> done
2024-10-15 17:10:21.612 [143921] main/101/interactive I> ready to accept requests
---
...

tarantool> box.space._schema:get('version')
---
- ['version', 1, 6, 8]
...

```